doc/whats_new.rst

```diff
@@ -41,6 +41,10 @@ New features
      Kullback-Leibler divergence and the Itakura-Saito divergence.
      By `Tom Dupre la Tour`_.
 
+   - Added :func:`metrics.regression.mean_squared_log_error`, a risk metric
```

Comment:
Added ..., which computes the mean of squares on the natural log of the error... or something like that...

The risk metric... seems unnecessary